test/test_shared_library.py

```diff
@@ -0,0 +1,255 @@
+    test_schema = json.loads(
+        open('translator/sample files/validation_schema/wzdx_v3.0_feed.json').read())
+    validate_write = translator_shared_library.validate_wzdx(
+        valid_wzdx_data, test_schema)
+    assert validate_write == True
+
+
+def test_validate_wzdx_invalid_wzdx_data():
```

Comment:
What's invalid here? Can we be more descriptive in the test name so we know what it is we are testing?

Reply:
i renamed the unit test name